### DISCUSSION TITLE
Block concurrent execution during job building

### DISCUSSION THREAD

**Discussion by MichalSznajder** [quartznet/quartznet#1826] [Q&A] [answered]
To block job concurrency `[DisallowConcurrentExecution]` attribute must be specified on class implementing `IJob` interface.

What is the reason it is not possible to block concurrent execution during job instance building? I imagine API similar to `JobBuilder.Create<MyJob>().DisallowConcurentExecution().Build()` ?

**Comment by lahma** [comment, marked as answer]
There should be `JobBuilder.Create<MyJob>().DisallowConcurrentExecution().Build()`, with to `r`s in Concurrent.

**Reply by MichalSznajder** [reply]
I was using too old version and this method was added in `3.5.0`.
